src/workflows/dags/data_warehouse.py

```
@@ -53,6 +53,14 @@ def extract_hourly_weather_data(**kwargs):
             days=7, **kwargs
         )
 
+        data = DataUtils.extract_data_from_bigquery(
```

Ruff (0.8.2)

Local variable `data` is assigned to but never used

Remove assignment to unused variable `data`

(F841)